index.html

```diff
@@ -32,42 +36,144 @@ <h1 class="logo">
         If you like what you see and have a project you need coded, don’t
         hesitate and contact me.
       </p>
-
-      <button type="button">Lets Connect</button>
-      <img class="design-one" src="bgbg.png" alt="hardwork bro" />
+      <button type="button" class="connect">Lets Connect</button>
+      <div class="background-image"></div>
     </section>
 
     <section class="Projects">
       <div class="headingtwo">Projects</div>
-      <img src="Rectangle21.png" alt="project 1" />
+
       <div class="project-info">
-        <div>Project name goes here</div>
-        <ul>
-          <li>HTML/CSS</li>
-          <li>Ruby on Rails</li>
-          <li>JavaScript</li>
+        <img class="thumbnails" src="Rectangle21.png" alt="project 1" />
+        <h3 class="project-name">AI blog writer</h3>
+        <ul class="project-details">
+          <li class="lang">HTML/CSS</li>
+          <li class="lang">Ruby on Rails</li>
+          <li class="lang">JavaScript</li>
+        </ul>
```

Comment:
- [ ] You did a good job on the styles but for this project, you are supposed to stick to the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9)design as much as possible. Please reduce the gap between project technologies and make sure it matches the design on the template.

### Your design

![image](https://user-images.githubusercontent.com/83803414/165037905-2c38e96b-5057-4878-98d9-c4f029cd2a76.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/165037960-0934c91d-e579-4061-8a3f-402e3e26a7e2.png)